tests/asyncio/test_grpc_helpers_async.py

```diff
@@ -522,11 +522,12 @@ def test_create_channel_explicit_with_duplicate_credentials():
     target = "example:443"
 
     with pytest.raises(exceptions.DuplicateCredentialArgs) as excinfo:
-        grpc_helpers_async.create_channel(
-            target,
-            credentials_file="credentials.json",
-            credentials=mock.sentinel.credentials,
-        )
+        with pytest.warns(DeprecationWarning):
```

Comment:
Add a comment to clarify that `credentials_file` is deprecated. Applies throughout